templates/bootstrap/index.tmpl

```diff
@@ -13,7 +13,7 @@
     <div class="navbar navbar-default navbar-static-top" role="navigation">
       <div class="container">
         <div class="navbar-header">
-          <a class="navbar-brand">Cucumberjs Report</a>
+          <a class="navbar-brand"></a>
```

Comment:
I agree for it to be taken from options if available else default it to Cucumberjs Report